front/app/modules/commercial/content_builder/admin/components/ContentBuilderEditModePreview/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { memo, useState } from 'react';
+
+// components
+import { Box, Icon, Button } from '@citizenlab/cl2-component-library';
+
+// styles
+import { stylingConsts, colors } from 'utils/styleUtils';
+
+// hooks
+import useLocale from 'hooks/useLocale';
+
+type ContentBuilderPreviewProps = {
+  projectId: string;
+};
+
+const ContentBuilderEditModePreview = React.forwardRef<
+  HTMLIFrameElement,
+  ContentBuilderPreviewProps
+>(({ projectId }, ref) => {
+  const locale = useLocale();
+  const [isMobile, setIsMobile] = useState(true);
+  const colorIfDesktopView = isMobile
+    ? colors.adminContentBackground
+    : colors.adminTextColor;
+  const colorIfMobileView = isMobile
+    ? colors.adminTextColor
+    : colors.adminContentBackground;
+  const buttonProps = {
+    height: '40px',
+    width: '92px',
+  };
+
+  return (
+    <Box
+      mt={`${stylingConsts.menuHeight + 20}px`}
+      minHeight={`calc(100vh - ${2 * stylingConsts.menuHeight + 20}px)`}
+      justifyContent="center"
+    >
+      <Box display="flex" flexDirection="column" alignItems="center">
```

Comment:
I see you are creating the tabs from scratch but perhaps Ben has reused an existing pattern in the app - could you check with him to make sure we are not re-implementing what's already there?